strawberryfields/backends/fockbackend/ops.py

```diff
@@ -260,8 +260,8 @@ def project_reset(modes, x, state, pure, n, trunc):
     Applies the operator :math:`\ket{00\dots 0}\bra{\mathbf{x}}` to the
     modes in `modes`.
     """
-    inSlice = sliceExp(modes, dict(zip(modes, x)), n)
-    outSlice = sliceExp(modes, dict(zip(modes, [0] * len(modes))), n)
+    inSlice = tuple(sliceExp(modes, dict(zip(modes, x)), n))
+    outSlice = tuple(sliceExp(modes, dict(zip(modes, [0] * len(modes))), n))
```

Comment:
What was the reason for this change?

Reply:
we were getting a deprecation warning during tests